config.yml

```diff
@@ -5,8 +5,8 @@ versions:
   rejson: 2.0.8
   redistimeseries: 1.6.9
   redisearch: 2.4.5
-  redisgraph: 2.8.12
-  redisbloom: 2.2.14
+  redisgraph: 2.8.13
+  redisbloom: 2.2.15
```

Comment:
do you need to update those modules versions manually whenever they get updated?

Reply:
Yup